Library/Formula/agda.rb

```diff
@@ -30,12 +36,6 @@ class Agda < Formula
 
   setup_ghc_compilers
 
-  head do
-    resource "stdlib" do
-      url "https://github.com/agda/agda-stdlib.git", :branch => "master"
-    end
-  end
-
   resource "stdlib" do
     url "https://github.com/agda/agda-stdlib.git", :branch => "2.4.2.4"
   end
```

Comment:
It seems this should be put inside stable do block otherwise it will overwrite the resource for head.